src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
```

Comment:
Please remove the redundant empty line after the class declaration as per the checklist.